modules/dicom_archive/php/NDB_Menu_Filter_dicom_archive.class.inc

```diff
@@ -180,6 +180,7 @@ class NDB_Menu_Filter_Dicom_Archive extends NDB_Menu_Filter
         }
         ksort($list_of_sites);
 
+        $this->tpl_data['Genders'] = array('M' => 'Male', 'F' => 'Female', 'O' => 'N/A');
```

Comment:
Okay, that makes sense.. but see my other comment. It doesn't need to come from PHP/smarty since, unlike Sites, it's not dynamic